### PR TITLE
fix: [PIE-4250]: added test cases for ServiceConfiguration and ServiceStudioDetails

### PR DESCRIPTION
**Pull request by vikrant-harness**
##### Summary:

added test cases for ServiceConfiguration and ServiceStudioDetails

##### Jira Links:

https://harness.atlassian.net/browse/PIE-4250

##### Screenshots:

Screenshot 2022-06-23 at 2 15 01 AM

Screenshot 2022-06-23 at 4 53 12 PM

You can use the following comments to re-trigger PR Checks

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress: `trigger cypress`
- Fix Prettier - `fix prettier`

### CODE REVIEW

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Comment by vikrant-harness** [comment]
retrigger titlecheck

**Review by satyam-harness — APPROVED** [review]

File: src/modules/75-cd/components/Services/ServiceStudio/ServiceConfiguration/__tests__/ServiceConfiguration.test.tsx [hunk 1]
```diff
@@ -0,0 +1,185 @@
...
+        </PipelineContext.Provider>
+      </TestWrapper>
+    )
+    const toggleYamlEl = getByText(container, 'YAML') as HTMLElement
+    expect(toggleYamlEl).toBeTruthy()
+    fireEvent.click(toggleYamlEl)
+    expect(container).toMatchSnapshot()
+    const toggleVisualEl = getByText(container, 'VISUAL') as HTMLElement
+    expect(toggleVisualEl).toBeTruthy()
+    // expect(container).toMatchSnapshot('new snapshot')
```
Comment (bhavya-sinha) [inline src/modules/75-cd/components/Services/ServiceStudio/ServiceConfiguration/__tests__/ServiceConfiguration.test.tsx:113]: commented code?

**Review by bhavya-sinha — APPROVED** [review]

File: src/modules/75-cd/components/Services/ServiceStudio/ServiceConfiguration/__tests__/ServiceConfiguration.test.tsx (hunk 1 above)
Comment (vikrant-harness) [reply src/modules/75-cd/components/Services/ServiceStudio/ServiceConfiguration/__tests__/ServiceConfiguration.test.tsx]: @bhavya-sinha removed it

**Review by satyam-harness — APPROVED** [review]

**Review by devinder-harness — APPROVED** [review]

**Review by bhavya-sinha — APPROVED** [review]

**Pull request merged**